Assets/Scripts/GameManager.cs

```diff
@@ -4,9 +4,26 @@
 
 public class GameManager : MonoBehaviour
 {
+    public GameObject Card2Prefab;
```

Comment:
For this and the rest, I feel like there is a better way to do this, but I don't know enough to say how. Would a single card prefab do the job rather than a whole bunch of GameObjects?

Reply:
I understand your concern as there is a fair amount of duplication.

Here were what I saw as the 3 options for how to do this: (I may have missed something)

1. We can give the Card all 52 images and let it do 52 case statements to pick the right card
2. We could give the GameManager 52 card prefabs that already had the correct image, but that would require 52 create statements (1 for each card)
3. We give the Card 4 Suit images that it knows how to swap based on it's suit, we then give the GameManager the 13 different values of cards and let it create 4 of each kind.